### PR TITLE
Add scripts for new message buffer

### PR DESCRIPTION
**Pull request by emma-miler**
Required for https://github.com/R2Northstar/NorthstarLauncher/pull/310

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
This works standalone without https://github.com/R2Northstar/NorthstarLauncher/pull/310, right? ^^

File: Northstar.Client/mod.json [hunk 1]
```diff
@@ -1,123 +1,123 @@
...
-		},
-		{
-			"Path": "ui/controller_prompts.nut",
-			"RunOn": "UI"
-		}
-	],
-	"Localisation": [
-		"resource/northstar_client_localisation_%language%.txt"
-	]
+  "Name": "Northstar.Client",
```
Comment (GeckoEidechse) [inline Northstar.Client/mod.json:2]: `Northstar.Client/mod.json` and `Northstar.CustomServers/mod.json` both have indentation/formatting changes that I assume where caused by running some auto-formatting and should be reverted. ^^

File: Northstar.Client/mod.json [hunk 2]
```diff
@@ -1,123 +1,123 @@
...
+  "Description": "Various ui and client changes to fix bugs and add better support for mods",
+  "Version": "1.9.7",
```
Comment (GeckoEidechse) [inline Northstar.Client/mod.json:4]: Similar version number shouldn't be changed in this PR.

File: Northstar.CustomServers/mod.json [hunk 3]
```diff
@@ -1,161 +1,167 @@
...
-			"Path": "/ai/_ai_turret_sentry.gnut",
-			"RunOn": "SERVER && MP",
-			"ServerCallback": {
-				"After": "AiTurretSentry_Init"
-			}
-		}
-	]
+  "Name": "Northstar.CustomServers",
+  "Description": "Attempts to recreate the behaviour of vanilla Titanfall 2 servers, as well as changing some scripts to allow better support for mods",
+  "Version": "1.9.7",
```
Comment (GeckoEidechse) [inline Northstar.CustomServers/mod.json:4]: Same indentation changes and version number change that should be reverted.

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

File: Northstar.CustomServers/mod/scripts/vscripts/messages_client.nut [hunk 4]
```diff
@@ -0,0 +1,16 @@
+untypeds
```
Comment (catornot) [inline Northstar.CustomServers/mod/scripts/vscripts/messages_client.nut:1]: would `untypeds` work?

Comment (uniboi) [reply Northstar.CustomServers/mod/scripts/vscripts/messages_client.nut:1]: no. This needs to be `untyped`

File: Northstar.Client/mod.json (hunk 1 above)
Comment (emma-miler) [reply Northstar.Client/mod.json:2]: pain

**Comment by emma-miler** [comment]
@GeckoEidechse fixed the issues ^^

File: Northstar.CustomServers/mod/scripts/vscripts/messages_server.nut [hunk 5]
```diff
@@ -0,0 +1,16 @@
+untyped
+globalize_all_functions
```
Comment (uniboi) [inline Northstar.CustomServers/mod/scripts/vscripts/messages_server.nut:2]:
```suggestion
global function MessageLoop_Init
```
There's no point in making `MessageLoop_Thread` global

**Review by uniboi — CHANGES_REQUESTED** [review]

File: Northstar.CustomServers/mod/scripts/vscripts/messages_server.nut (hunk 5 above)
Comment (emma-miler) [reply Northstar.CustomServers/mod/scripts/vscripts/messages_server.nut]: This is a bit inconsequential

**Comment by emma-miler** [comment]
Update: this PR is now obsolete due to a revision in the way the original PR works

**Pull request closed without merging**